src/wp-login.php

```diff
@@ -314,6 +314,15 @@ function login_footer( $input_id = '' ) {
 	<?php
 	$languages = get_available_languages();
 
+	/**
+	 * Filters the Languages select input activation on the login screen.
+	 *
+	 * @since 5.9.0
+	 *
+	 * @param bool Whether to display the Languages select input on the login screen.
+	 */
+	$display_login_language_dropdown = apply_filters( 'display_login_language_dropdown', true );
```

Comment:
This does nothing since the `$display_login_language_dropdown` is not used. If `false`, `get_available_languages()` shouldn't be called either.

Reply:
Wired the filter in https://github.com/WordPress/wordpress-develop/pull/2074/commits/b0b04a9fb462d67dbb9aa1b23e30dccb10645189.